src/index.js

```diff
@@ -28,7 +28,7 @@ function getClosestPackage(modulePath) {
   // Catch findRoot or require errors
   try {
     root = findRoot(modulePath);
```

Comment:
If we are creating this for ourselves, thoughts on dropping this fineRoute dependency and use `process.cwd()` like we do in our default webpack configurations for determining the root path?

Reply:
I don't think it can be as simple as that. We need to determine the package roots of each individual package in the bundle, not just the overall package. And given that we can't guarantee much about any projects individual structure, we need to do some recursion to find the closest package.json file for which the resource is defined. 

findRoot does this for us. It's not terribly complicated logic, so if the dependency becomes a problem down the line I'm not to worried about removing it. But as is, it's tested and functional, so I'm fairly comfortable with it.